### ISSUE TITLE
dead/old link in foxx-manager help

### ISSUE THREAD

**Issue by Thijxx** [arangodb/arangodb#2441]
## my environment running ArangoDB
- [X] 3.0

On this operating system:
  - [X] other: CentOS7

### Foxx
```
Use  foxx-manager --help  to show a list of global options

There is also an online manual available at:
https://docs.arangodb.com/Foxx/Install/
```
This link does not exist. It forwards to:` https://docs.arangodb.com/3.1/Manual/ `and that is in my case not the version I use.

These are the steps to reproduce:
1) run foxx-manager help
2) click the url

**Comment by jsteemann** [comment]
Just fixed in 3.0 and 3.1. devel will follow soon once we merge back some branch.

**Comment by dothebart** [comment]
3.1.19 is available for download - closing.
